pipeline/backend/docker/docker.go

```diff
@@ -232,3 +233,10 @@ var (
 		Timestamps: false,
 	}
 )
+
+func isErrContainerNotFoundOrNotRunning(err error) bool {
```

Comment:
it's ok to filter them out now ... but that's an open TODO ... since we should finde the root issue

Reply:
This is somehow expected. For example if we start a container with the command `exit 1` Kill will always run into a `container is not running issue`.